tests/test_operation.py

```diff
@@ -1271,7 +1271,9 @@ def test_basis_state_decomposition(self, monkeypatch):
 
         # We have to patch BasisStatePreparation where it is loaded
         monkeypatch.setattr(
-            qml.ops.qubit, "BasisStatePreparation", lambda *args: call_args.append(args)
+            qml.ops.qubit.state_preparation,
```

Comment:
How come these were changed? Also seemed like some parts of the ops were reformatted, though with `black -l 100` no changes were applied. :thinking:

Reply:
`qml.ops.qubit` goes to `qml.ops.qubit.state_preparation`.

The path to the operations changed. As the comment says "we have to patch BasisStatePreparation where it is loaded". Don't fully understand how it works, but this gets the test to pass.

Reply:
Oh boy, right!